### PR TITLE
Fix secondary description not written to language file

### PR DESCRIPTION
**Pull request by MuXiu1997**
fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/9989

### CODE REVIEW

**Comment by MuXiu1997** [comment]
Extracting the interface may cause the secondary description of other mods to fail

**Comment by MuXiu1997** [comment]
fix GTNewHorizons/GT-New-Horizons-Modpack#9989

File: src/main/java/gregtech/api/metatileentity/MetaTileEntity.java [hunk 1]
```diff
@@ -107,14 +107,6 @@ public MetaTileEntity(String aName, int aInvSlotCount) {
         mName = aName;
     }
 
-    /**
-     * This method will only be called on client side
-     * @return whether the secondary description should be display. default is false
-     */
-    public boolean isDisplaySecondaryDescription() {
```
Comment (glowredman) [inline src/main/java/gregtech/api/metatileentity/MetaTileEntity.java:114]: Do not just remove public methods especially not in the api package. Just annotate the method as `@Deprecated`

Comment (MuXiu1997) [reply src/main/java/gregtech/api/metatileentity/MetaTileEntity.java:114]: got it

**Review by glowredman — APPROVED** [review]
Seems alright

**Pull request merged**